sickbeard/clients/generic.py

```diff
@@ -150,21 +150,21 @@ def _get_torrent_hash(result):
                 result.hash = b16encode(b32decode(result.hash)).lower()
         else:
             if not result.content:
-                logger.log(u'Torrent without content', logger.ERROR)
-                raise Exception('Torrent without content')
+                logger.log(u'Torrent without content', logger.WARNING)
 
             try:
                 torrent_bdecode = bdecode(result.content)
-            except BTFailure:
-                logger.log(u'Unable to bdecode torrent', logger.ERROR)
-                logger.log(u'Torrent bencoded data: %r' % result.content, logger.DEBUG)
-                raise
-            try:
                 info = torrent_bdecode["info"]
+                result.hash = sha1(bencode(info)).hexdigest()
+            except BTFailure:
+                logger.log(u'Unable to bdecode torrent. Probably torrent was removed from tracker', logger.WARNING)
```

Comment:
"Unable to bdecode torrent. Invalid torrent"

It is removed becouse it is invalid not invalid becouse it got removed. BitTorrent is descentralized